Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3142,18 +2045,19 @@ private void setActivityLifeCycleObserver(Application application) {
 
         @Override
         public void onActivityCreated(Activity activity, Bundle bundle) {
+            intentState_ = INTENT_STATE.PENDING;
             if (BranchViewHandler.getInstance().isInstallOrOpenBranchViewPending(activity.getApplicationContext())) {
                 BranchViewHandler.getInstance().showPendingBranchView(activity);
             }
         }
 
         @Override
         public void onActivityStarted(Activity activity) {
+            intentState_ = INTENT_STATE.PENDING;
```

Comment:
Doesn't this mean that we won't do any initialization until onResume is called now? I thought we discussed just doing this in the scenario where we determine the intent has stale data?